modules/multi-architecture-import-imagestreams.adoc

```diff
@@ -0,0 +1,72 @@
+//Module included in the following assemblies
+//
+//post_installation_configuration/cluster-tasks.adoc
+
+:_content-type: PROCEDURE
+[id="multi-architecture-import-imagestreams_{context}"]
+
+= Importing manifest lists in image streams on your multi-architecture cluster 
+
+On a {product-title} {product-version} multi-architecture cluster, the image streams in the cluster do not import manifest lists automatically. You must manually change the default `importMode` option to the `PreserveOriginal` option in order to import the manifest list.
+
+[NOTE]
+====
+For this procedure to run successfully, the `referencePolicy.type` field of your `ImageStream` object must be set to the `Source` type. 
```

Comment:
Do you think this should be an important since users have to do this in order for this to work?

Also, what do you think of rewording to:
```suggestion
The `referencePolicy.type` field of your `ImageStream` object must be set to the `Source` type for this procedure to run successfully. 
```
because I think with "For this procedure to run successfully" first makes it passive in the form of be + past participate (active/passive voice isn't my strong suit so I could be wrong).

Reply:
Thats a great point, I'll change it to Important. 

Hmmm I'm not sure if its passive, it is tough to tell bc the past participle of the present "run" is also "run" lol. 
But either way I like your suggestion of the sentence better, thank you!